index.js

```diff
@@ -304,20 +304,31 @@ export const Classes = {
 		selected: 'selected'
 	},
 	outcomes: {
+		assessed: 'assessed',
+		demonstration: 'demonstration',
+		demonstratableLevel: 'demonstratable-level',
 		intent: 'intent',
 		intents: 'intents',
 		intentList: 'intent-list',
+		loa: 'level-of-achievment',
 		organizationIntentList: 'organization-intent-list',
 		outcome: 'outcome',
 		outcomes: 'outcomes',
 		selected: 'selected',
-		suggested: 'suggested',
-		demonstratableLevel: 'demonstratable-level'
+		suggested: 'suggested'
 	},
 	meetings: {
 		bookable: 'bookable'
 	},
-	relativeUri: 'relative-uri'
+	relativeUri: 'relative-uri',
+	userProgress: {
+		outcomes: {
+			root: 'user-progress-outcomes',
+			outcome: 'user-progress-outcome',
+			activities: 'outcome-activities',
```

Comment:
Should this be either moved up to the more general `outcomes` class or else the value modified to be `user-progress-outcome-activities`?

Reply:
The latter

Reply:
```suggestion
			activities: 'user-progress-outcome-activities',
```